catalyst/core/callbacks/checkpoint.py

```diff
@@ -199,6 +199,11 @@ def on_exception(self, runner: IRunner):
         if not utils.is_exception(exception):
             return
 
+        if runner.device.type == "xla":
```

Comment:
could we define it as global `MODEL_SAVE_FN`?

Reply:
What do you think about `save_fn` as runner function?